qiskit_aer/backends/name_mapping.py

```diff
@@ -250,33 +267,79 @@ def __init__(self, num_ctrl_qubits, ctrl_state=None):
             None,
             num_ctrl_qubits,
             ctrl_state=ctrl_state,
-            base_gate=SwapGate(),
+            base_gate=CSwapGate(),
         )
 
 
 PHI = Parameter("phi")
 LAM = Parameter("lam")
 NAME_MAPPING = {
+    "u1": U1Gate,
+    "u2": U2Gate,
+    "u3": U3Gate,
+    "u": UGate,
+    "p": PhaseGate,
+    "r": RGate,
+    "rx": RXGate,
+    "rxx": RXXGate,
+    "ry": RYGate,
+    "ryy": RYYGate,
+    "rz": RZGate,
+    "rzz": RZZGate,
+    "rzx": RZXGate,
+    "id": IGate,
+    "h": HGate,
+    "x": XGate,
+    "y": YGate,
+    "z": ZGate,
+    "s": SGate,
+    "sdg": SdgGate,
+    "sx": SXGate,
+    "sxdg": SXdgGate,
+    "t": TGate,
+    "tdg": TdgGate,
+    "swap": SwapGate,
+    "cx": CXGate,
+    "cy": CYGate,
+    "cz": CZGate,
+    "ccx": CCXGate,
+    "csx": CSXGate,
+    "cp": CPhaseGate,
+    "cu": CUGate,
+    "cu1": CU1Gate,
```

Comment:
For the standard gates they're covered by default from https://github.com/Qiskit/qiskit/blob/main/qiskit/circuit/library/standard_gates/__init__.py#L49 the custom name mapping passed to `convert_to_target()` is additive, the code takes `get_standard_gate_name_mapping()`'s output and runs `.update(NAME_MAPPING)` on it. Was there a reason you had to add these here too? Also in general if you pass in a gate class instead of an instance like this those instructions will be treated as variadic instructions which doesn't seem correct for standard library gates.

Reply:
removed standard gates from NAME_MAPPING